apps/artemis-web/components/hijack-info/hijack-info.tsx

```diff
@@ -29,17 +31,51 @@ class HijackInfoComponent extends Component<any, any> {
     this.setOpenModalState = props.setOpenModalState;
     this.hijackKey = props.hijackKey;
 
+    const [hijackInfoLeft, hijackInfoRight] = extractHijackInfos(
+      this.props.hijackDataState,
+      {
+        tooltips: this.props.tooltips,
+        setTooltips: this.props.setTooltips,
+        context: this.props.context,
+      }
+    );
+
+    this.hijackInfoLeft = hijackInfoLeft;
+    this.hijackInfoRight = hijackInfoRight;
+
     this.state = {
       seenState: false,
       withdrawState: false,
       editComment: false,
       commentSuccess: true,
+      gripState: false,
+      event_id : ""
     };
 
     this.isMobile = props.isMobile;
 
     this.commentRef = React.createRef();
     this.selectRef = React.createRef();
+
+  }
+
+  async componentDidMount() {
+
+    const asn = this.hijackInfoLeft["Hijacker AS:"][0].props.asn;
+    const prefix = this.hijackInfoLeft["Prefix"][0];
```

Comment:
what about the hijack type (which should be GRIP-mapped) according to comment https://github.com/FORTH-ICS-INSPIRE/artemis-web/issues/114#issuecomment-884239572 ?

Reply:
fixed!